pages/common/act.md

```diff
@@ -0,0 +1,28 @@
+# act
+
+> Execute GitHub Actions locally using Docker.
+> More information: <https://github.com/nektos/act>.
+
+- List the available actions:
+
+`act -l`
+
+- Run the default event:
+
+`act`
+
+- Run a specific event:
+
+`act {{event_type}}`
+
+- Run a specific action:
+
+`act -a {{action_id}}`
+
+- Do not actually run the actions (dry-run):
```

Comment:
Nice! IIRC GitHub actions are only just being released.

Perhaps this will read better?
```suggestion
- Do not actually run the actions (i.e a dry run):
```

Reply:
Yeah! I've had a good play with them, and created a couple of actions myself. Pretty nice. However, I'm not sure `act` supports the new YAML syntax yet. But, afaik GitHub are going to release their own version later this year that allows you to run the new YAML-syntax workflows locally.

I'll make that change shortly, thanks!